src/snaskii.c

```diff
@@ -108,7 +108,7 @@ snake_t snake; /* The snake instance. */
 
 /* Energy blocks. */
 
-struct {
+struct _energy_block {
```

Comment:
This looks out of the codestyle. It would be better if you name it `energy_block`